src/main/webapp/dataset.xhtml

```diff
@@ -144,41 +145,61 @@
                                                     <button type="button" class="btn btn-primary btn-access-dataset dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                                         #{bundle['dataset.accessBtn']} <span class="caret"></span>
                                                     </button>
-                                                    <ul class="dropdown-menu pull-right text-left">
-
+                                                    <ul class="dropdown-menu pull-right text-left" style="padding-left:1em; padding-right:1em;" >
                                                         <!-- DOWNLOAD -->
-                                                        <ui:fragment rendered="#{DatasetPage.canDownloadFiles()}">
+                                                        <ui:fragment rendered="#{DatasetPage.canDownloadFiles()  and !(DatasetPage.tooLargeToDownload) }">
                                                             <li class="dropdown-header">#{bundle['dataset.accessBtn.header.download']} <span class="glyphicon glyphicon-download-alt"/></li>
                                                             <!-- NORMAL DOWNLOAD BUTTON (NO TABULAR FILES) -->
-                                                            <li jsf:rendered="#{!DatasetPage.versionHasTabular}">
-                                                                <p:commandLink update="@form" actionListener="#{DatasetPage.validateAllFilesForDownloadArchival()}" styleClass="btn-download">
+                                                            <li jsf:rendered="#{!DatasetPage.versionHasTabular and !(DatasetPage.tooLargeToDownload)}">                 
```

Comment:
isn't the 2nd part of the rendered always true? (since it's in the ui fragment)

Reply:
The difference here is that we render different command links depending on whether there are tabular files in the dataset - line 153 for no tabular files and 164 below for has tabular files.

Reply:
I get that, but that explains the tabular check on the renders of the li tags. But I'm talking about the ui:fragment surrounding these li tags. Let me know if it's easier to discuss on a call, viewing the code.

Reply:
Ok I get what you're saying now. I can remove it.